src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -105,10 +111,32 @@
     return this;
   }
 
+  @Override
+  public Span start() {
+    return startManual();
+  }
+
+  @Override
+  public ActiveSpan startActive() {
+    if (!ignoreActiveSpan) {
```

Comment:
this property itself hints maybe the ActiveSpan thing should be an extension, not a requirement